src/main/java/teammates/common/util/Config.java

```diff
@@ -103,53 +98,113 @@ public final class Config {
     /** The value of the "app.maintenance" in build.properties file. */
     public static final boolean MAINTENANCE;
 
+    // The following properties are not used in production server.
+    // So they will only be read from build-dev.properties file.
```

Comment:
Similarly for this. With this said, the reordering of the properties may not be necessary.